src/chatgpt-api.ts

```diff
@@ -335,11 +335,9 @@ export class ChatGPTAPI {
     const systemMessageOffset = messages.length
     let nextMessages = messages.concat([
       {
-        ...{
-          role: 'user',
-          content: text,
-          name: opts.name
-        }
+        role: 'user',
```

Comment:
the reason I was doing the weird destructuring here is because if `name` is undefined, it will remove it from the output instead of keeping a `"name": undefined` in the JSON